tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -448,17 +448,17 @@ struct bt_le_per_adv_param;
 struct bt_le_per_adv_sync_param;
 struct bt_le_adv_param;
 struct bt_data;
-struct bt_le_ext_adv *tester_gap_ext_adv_get(void);
+struct bt_le_ext_adv *tester_gap_ext_adv_get(uint8_t ext_adv_id);
 struct bt_le_per_adv_sync *tester_gap_padv_get(void);
-int tester_gap_create_adv_instance(struct bt_le_adv_param *param, uint8_t own_addr_type,
-				   const struct bt_data *ad, size_t ad_len,
+int tester_gap_create_adv_instance(uint8_t ext_adv_id, struct bt_le_adv_param *param,
```

Comment:
One thing that concerns me is the name of the new parameter here. 

In the LE Audio files it's a "source_id", here it is a "ext_adv_id", in the definitions of the functions it is a "ext_adv_idx" and then param->sid (Set ID) is directly assigned to this. 

I suggest to clean up the naming and be more consistent and clear. 
Perhaps use `sid` everywhere, and then treat it as a `id` and not an `index`? Alternatively, we can keep it as an index value and name it thus if you want to use it to do direct lookups in an array